src/Template/Aefis/xml/e2b.ctp

```diff
@@ -229,7 +233,7 @@
                     ?>                 
                 </primarysourcereaction>
                 <reactionmeddraversionllt>WHO-ART</reactionmeddraversionllt>
-                <reactionmeddrallt><?php echo $aefi['description_of_reaction']; ?></reactionmeddrallt>
+                <reactionmeddrallt><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['description_of_reaction']); ?></reactionmeddrallt>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate